docker/dev/frontend/Dockerfile

```diff
@@ -8,7 +8,13 @@ ENV USER=dev
 
 RUN npm i -g npm
 
-RUN usermod -l $USER -d /home/$USER -m node
+# `--no-log-init` is required as a workaround to avoid disk exhaustion.
+#
+# Read more at:
+# * https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user
+# * https://github.com/golang/go/issues/13548
+RUN useradd --no-log-init -d /home/$USER -m $USER -g node
```

Comment:
Unfortunately `--no-log-init` does not exists for `usermod`. So we've to use `useradd --no-log-init` to create a user and avoid the nasty bug for subsequent calls to `usermod`.

Reply:
This is interesting and hilarious. Thanks for that information.

Reply:
Would be great to run CI/CD pipeline to check that this will not break anything. Tried running specs locally but some turned red and I'm not sure if it's because of my machine or this change is breaking something.

Reply:
No this is a local dev dockerfile, only used for development